### PR TITLE
creating basic features

### PR DESCRIPTION
**Pull request by nimplay**
- [ ] Webpack configuration
- [ ] linters configuration
- [ ] index Js
- [ ] index HTML
- [ ] Style css
- [ ] add itens into the score box
- [ ] basic logic
- [ ] readme file

### CODE REVIEW

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Pull Request :red_circle:

I know you have made a great effort in your project :blush: but unfortunately, I cannot proceed review of your pull request.

- [ ] You don't have a summary in your pull request. Kindly add it :+1:
- [ ] You don't have a descriptive readme file in your project. Please add it :blush:
- [ ] You didn't add the linters properly. Kindly add them properly according to the [instructions](https://github.com/microverseinc/linters-config/tree/master/html-css-js) :memo:

**_This won't be counted as a review because your pull request is invalid :red_circle:_**

**Comment by nimplay** [comment]
Summary:
the linters were corrected the readme file was added and the branches created correctly

File: src/index.html [hunk 1]
```diff
@@ -4,10 +4,26 @@
...
+        <button class="refresh-buttom">Refresh</button>
+      </div>
+      <ul class="score-table">
+      </ul>
+    </div>
+    <form class="form">
+      <h2 class="form-title">Add Your Score</h2>
+      <input type="text" placeholder="Your Name" class="name-box">
+      <input type="text" placeholder="Your Score" class="score-box">
+      <button type="button" class="submit-button">Submit</button>
```
Comment (zieeco) [inline src/index.html:25]: - kindly ensure the placement and width of your button match the wireframe. please, fix this issue.

button


frame

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @nimplay,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Project setup correctly ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nimplay** [comment]
the corrections of the previous reviewer were applied

**Review by omarsalem7 — APPROVED** [review]
Hi @nimplay 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**